### PR TITLE
[CodeAnalysis] EasyCodingStandard added

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -374,6 +374,7 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 * [Better Reflection](https://github.com/Roave/BetterReflection) - AST-based reflection library that allows analysis and manipulation of code
 * [Code Climate](https://codeclimate.com) - An automated code review.
 * [Dissect](https://github.com/jakubledl/dissect) - A set of tools for lexical and syntactical analysis.
+* [EasyCodingStandard](https://github.com/Symplify/EasyCodingStandard) - Best intro tool for Coding Standards. Combine PHP_CodeSniffe and PHP-CS-Fixer.
```
Comment (a-komarev) [inline README.md]: PHP_CodeSniffe > PHP_CodeSniffer

Comment (TomasVotruba) [inline README.md]: Good catch, thank you

**Comment by TomasVotruba** [comment]
I fixed the typo

**Comment by TomasVotruba** [comment]
Ping

**Comment by TomasVotruba** [comment]
What needs to be done here? Just let me know

**Comment by TomasVotruba** [comment]
Ping

**Pull request closed without merging**